dbsc/workflow/rules/freesurfer.smk

```diff
@@ -1,69 +1,73 @@
-from os.path import join 
+from os.path import join
+
+# Directories
+freesurfer_dir = join(config["out_dir"], "freesurfer")
+
+# Freesurfer references (with additional in rules as necessary)
+# B. Fischl, A. van der Kouwe, C. Destrieux, E. Halgren, F. Ségonne, D.H. Salat, E. Busa, L.J. Seidman, J. Goldstein, D. Kennedy, V. Caviness, N. Makris, B. Rosen, A.M. Dale. Automatically parcellating the human cerebral cortex. Cereb. Cortex, 14 (2004), pp. 11-22, 10.1093/cercor/bhg087
+
 
 rule thalamic_segmentation:
+    """Perform thalamus segmentation
+
+    Reference: J.E. Iglesias, R. Insausti, G. Lerma-Usabiaga, M. Bocchetta, K. Van Leemput, D.N. Greve, A. van der Kouwe, B. Fischl, C. Caballero-Gaudes, P.M. Paz-Alonso. A probabilistic atlas of the human thalamic nuclei combining ex vivo MRI and histology. NeuroImage, 183 (2018), pp. 314-326, 10.1016/j.neuroimage.2018.08.012
+
+    NOTE: Output file is defined by Freesurfer script
+    """
     input:
-        freesurfer_dir=join(config['bids_dir'], 'derivatives/freesurfer'),
+        freesurfer_dir=freesurfer_dir,
     params:
         fs_license=config["fs_license"],
-        threads=workflow.cores,
     output:
-        thal_seg=bids(
-            root=join(config["output_dir"], "freesurfer"),
-            datatype="anat",
-            suffix="T1.mgz"
-            space="Freesurfer",
-            **config["subj_wildcards"],
-        ),
-    container: 
-        config['singularity']['freesurfer']
+        thal_seg=join(freesurfer_dir, "{subject}/mri/ThalamicNuclei.v12.T1.mgz"),
+    threads: workflow.cores
+    container:
+        config["singularity"]["freesurfer"]
     shell:
         "FS_LICENSE={params.fs_license} "
         "ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS={params.threads} "
         "SUBJECTS_DIR={input.freesurfer_dir} "
         "segmentThalamicNuclei.sh {{subject}}"
```

Comment:
I don't know what's in this script, but just to double check: Do you want to pass it the string `{subject}`? Otherwise you may want `{wildcards.subject}`